kube/metrics.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package kube
+
+const (
+	// Hostinfo App values for the Tailscale Kubernetes Operator components.
+	AppOperator        = "k8s-operator"
+	AppAPIServerProxy  = "k8s-operator-proxy"
+	AppIngressProxy    = "k8s-operator-ingress-proxy"
+	AppIngressResource = "k8s-operator-ingress-resource"
+	AppEgressProxy     = "k8s-operator-egress-proxy"
+	AppConnector       = "k8s-operator-connector-resource"
+
+	// Clientmetrics for Tailscale Kubernetes Operator components
+	IngressProxyCount              = "k8s_ingress_proxies"   // L3
+	IngressResourceCount           = "k8s_ingress_resources" // L7
+	EgressProxyCount               = "k8s_egress_proxies"
+	ConnectorResourceCount         = "k8s_connector_resources"
+	ConnectorWithSubnetRouterCount = "k8s_connector_subnetrouter_resources"
+	ConnectorWithExitNodeCount     = "k8s_connector_exitnode_resources"
+	NameserverCount                = "k8s_nameserver_resources"
+)
```

Comment:
The client metrics are not new in this PR, just moved them around to make it easier to see all metric types for operator components

Reply:
Maybe prefix them with Metric like you prefix App on the other ones